### DISCUSSION TITLE
"other_thumb" not respected when using "ball_side": "left"

### DISCUSSION THREAD

**Discussion by BansheeHero** [joshreve/dactyl-keyboard#75] [Q&A]
It works with trackball options so the impact is very low.
User can always work around it by having 2 configs.

When using a JSON like this:
```json
    "ball_side": "left",
    "thumb_style": "MINI",
    "other_thumb": "MINIDOX"
```
I would expect the sides to be different even if there is no trackball. Instead both sides follow thumb_style.
Is that something you decided against?

Tried both engines on master.

**Comment by joshreve** [comment]
Interesting.  That is definitely a bug.  I was focused on the trackball for that specific parameter, but it should theoretically work.  I will take a look when I can.  I'm guessing I only put the conditional under the trackball section.
